src/GSoft.Extensions.MediatR/GSoft.Extensions.MediatR.csproj

```diff
@@ -10,17 +10,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="MediatR" Version="11.1.0" />
-    <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />
+    <PackageReference Include="MediatR" Version="12.0.1" />
+    <PackageReference Include="MediatR.Contracts" Version="2.0.1" />
```

Comment:
I discovered that this package is manually published by jbogard, and he already made a mistake two weeks ago.

Although it's implicitely referenced by `MediatR` package, the `MediatR` still referenced the bugged and unlisted previous version `MediatR.Contracts`, so we need to explicitely reference `MediatR.Contracts` to better control our dependencies.